platform/windows/display_server_windows.cpp

```diff
@@ -1766,15 +1766,15 @@ void DisplayServerWindows::window_set_vsync_mode(DisplayServer::VSyncMode p_vsyn
 	_THREAD_SAFE_METHOD_
 #if defined(VULKAN_ENABLED)
 	// TODO disabling for now
-	//context_vulkan->set_vsync_mode(p_window, p_vsync_mode);
+	// context_vulkan->set_vsync_mode(p_window, p_vsync_mode);
```

Comment:
That's weird, `clang-format` is configured specifically not to do that. Did you author this on an old branch (pre #54133) before rebasing on `master` maybe?

Reply:
Oh no, sorry, that's my fault. I had to comment out `SpacesInLineCommentPrefix` from .clang-format because that's a new Clang 13 option and LLVM haven't yet updated Windows clang-format binary to the new version. I thought it was okay to add a leading space, though, since vast majority of comments in those files already use a leading space and it was just more consistent. I can undo those changes, though, if that would be better.